### PR TITLE
Return CSR to use api v1beta1

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: pkg/controller/cluster/csr.go [hunk 1]
```diff
@@ -102,14 +103,14 @@ func (c *Controller) createCertificateSigningRequest(ctx context.Context, labels
 			},
 		},
 		Spec: certificates.CertificateSigningRequestSpec{
-			SignerName: csrSignerName,
-			Request:    encodedBytes,
-			Groups:     []string{"system:authenticated", "system:nodes"},
-			Usages:     csrKeyUsage,
+			//SignerName: csrSignerName,
```
Comment (harshavardhana) [inline pkg/controller/cluster/csr.go:106]: why are all these commented?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by Alevsk — CHANGES_REQUESTED** [review]
testing all this

File: pkg/controller/cluster/csr.go (hunk 1 above)
Comment (dvaldivia) [reply pkg/controller/cluster/csr.go]: done

**Review by Alevsk — APPROVED** [review]
Tested on 1.17 - 1.22 k8s

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**